### PR TITLE
Food filter

### PR DESCRIPTION
**Pull request by NZenitram**
Food and Exercise filters up

### CODE REVIEW

**Comment by NZenitram** [comment]
Form field errors fixed -- Exercise and food filtering implemented but tests aren't passing.

File: foods.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h3 id="manage-foods-header">Manage Foods</h3>
 
   <div id="foods" class='fe-table-div'>
     <div id="foods-filter" class="filter">
-      <p><b>Filter by Name</b></p> <input class="form-input" type="text" name="foods-filter">
+      <p><b>Filter by Name</b></p> <input id="food-filter" class="form-input" type="text" name="foods-filter">
```
Comment (JStans12) [inline foods.html:27]: filters baby

File: lib/index.js [hunk 2]
```diff
@@ -89,6 +106,24 @@ function populateExercises(){
...
+
+   var input = document.getElementById('exercise-filter');
+   var filter = input.value.toUpperCase();
+   var table = document.getElementById('exercises-table');
+   var tr = table.getElementsByTagName("tr");
+   for (i = 0; i < tr.length; i++) {
+     td = tr[i].getElementsByTagName("td")[0];
+     if (td) {
+       if (td.innerHTML.toUpperCase().indexOf(filter) > -1) {
+         tr[i].style.display = "";
```
Comment (JStans12) [inline lib/index.js:119]: what is this wizardry?

File: test/user_can_filter_foods.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    calories.sendKeys('500');
+    addFoodButton.click()
+    driver.sleep(1000);
+
+    name.sendKeys('lifting');
+    calories.sendKeys('500');
+    addFoodButton.click()
+    driver.sleep(1000);
+
+    name.sendKeys('balling out');
```
Comment (JStans12) [inline test/user_can_filter_foods.js:39]: ball so hard

**Pull request merged**